### PR TITLE
Create a script that runs two Foundry networks to use it in the ics scenario

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

**Comment by junha1** [comment]
This is the only script that will be tested as PoC?

File: ibc.ts/src/scenario/runChains.ts [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        ["-c", "./chainB.schem.json", "--config", "./chainB.config.toml"],
+        {
+            cwd: "./chainB"
+        }
+    );
+    streamOutToDebug(foundryProcess);
+}
+
+async function checkChainAAndBAreRunning() {
+    // Wait for Foundry to listen on the port, three seconds is a random value.
```
Comment (junha1) [inline ibc.ts/src/scenario/runChains.ts]: a random value -> just an arbitrary value?

Comment (majecty) [reply ibc.ts/src/scenario/runChains.ts]: Good. I'll fix it to "Wait for Foundry to listen on the port; three seconds is an arbitrary value."

Comment (majecty) [reply ibc.ts/src/scenario/runChains.ts]: @junha1 I fixed it.

**Pull request merged**